admin/users/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^$', views.UserFormView.as_view(),
+        name='search'),
+    url(r'^id-(?P<guid>[a-z0-9]+)/$', views.UserView.as_view(),
+        name='user'),
+    url(r'^id-(?P<guid>[a-z0-9]+)/rem-2-factor/$', views.remove_2_factor,
```

Comment:
Currently, it undeletes their old model instance, and then repopulates the cleared fields when they re-enter the 2F verification code. Simplest solution would be to not clear on `delete`, only disable the addon, and then clear when a user re-authorizes but not when an admin does. See [this code](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/addons/twofactor/models.py#L51-L61). 

I don't know of any security threats this approach could pose, but I'm also not as intimately familiar with the two factor addon as I am with most others. May want to run this by Steve.